include/baresip.h

```diff
@@ -855,7 +856,7 @@ int  ua_disable_autoanswer(struct ua *ua, enum answer_method met);
 int  ua_call_alloc(struct call **callp, struct ua *ua,
 		   enum vidmode vidmode, const struct sip_msg *msg,
 		   struct call *xcall, const char *local_uri,
-		   bool use_rtp);
+		   bool use_rtp, const char *user_data);
```

Comment:
Can't you use a setter function for the user_data?
You had already one, why you removed it?

Reply:
Looking through the code it felt more natural to me to set the user_data when the call object is created.

Reply:
I've changed the code to use a setter instead.